rootfs/scripts/assign_frequencies.py

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env python3
 import sys
+import os
+import shutil
 import argparse
+import fileinput
 from pprint import pprint
 
+if os.getenv("SPAM", default=False):
+    servicesd_path = "/Users/fred/services.d/"
```

Comment:
Should these debugging artifacts remain in the production container?

I don't mind either way, just flagging it in case of oversight... :)

Reply:
I've left these debug artifacts in a few places to help me run it outside of docker. I usually don't hard code paths though and pass it in via ENV...

Reply:
No probs